core/src/cnf/mod.rs

```diff
@@ -1,5 +1,27 @@
 use once_cell::sync::Lazy;
 
+#[macro_export]
+macro_rules! lazy_env_parse {
+	($key:expr, $t:ty, $default:expr) => {
+		once_cell::sync::Lazy::new(|| {
+			std::env::var($key)
+				.and_then(|s| Ok(s.parse::<$t>().unwrap_or($default)))
+				.unwrap_or($default)
+		})
+	};
+}
+
+#[macro_export]
+macro_rules! lazy_env_parse_or_else {
+	($key:expr, $t:ty, $default:expr) => {
+		once_cell::sync::Lazy::new(|| {
+			std::env::var($key)
+				.and_then(|s| Ok(s.parse::<$t>().unwrap_or_else($default)))
+				.unwrap_or_else($default)
+		})
+	};
+}
+
```

Comment:
@emmanuel-keller could we place these into the `core/src/mac/mod.rs` file? It stores globally used macros...

Reply:
Sure! And added some documentation